.github/workflows/nubots.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
         run: ./b build
 
       - name: Test the code
-        run: ./b tests run
+        run: for i in {1..20}; do ./b tests run Kine; done
```

Comment:
What is this I am confused

Reply:
This is me trying to debug the segfaults on the tests which I can't replicate on my machine :sweat:

Reply:
Ahh okay that sucks 😓 can I help in anyway?

Reply:
If you can replicate the segfault on your machine, that would be a huge help! My machine doesn't segfault, but adding this line [here](https://github.com/NUbots/NUbots/pull/728/commits/db3f4ed34d7a40f7d2394b819259f5374d7357e8) seems to introduce it. I don't know why though. Fixing the cause is the goal

Reply:
Looks like it was a reference issue which has been fixed by explicit `std::move` in the `unique_ptr` construction.

Reply:
I have stopped using `unique_ptr` at all because it seemed like that problem wasn't actually fixed by that change. This change seems to have fixed things... _here's hoping_